doc/development/deprecations.rst

```diff
@@ -9,6 +9,11 @@ deprecations are listed below.
 Pending deprecations
 --------------------
 
+* The ``control_wires`` argument in the ``qml.ControlledOperation`` class is deprecated. Instead, use the ``control`` argument.
```

Comment:
I thought there was just going to be `mat` and `wires`? What is the `control` argument?

Reply:
Good catch😂